From 08bfdac1979d6d94ff341be6cfa8df98e712615a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 19:07:46 +0300
Subject: [PATCH] Makefile: don't save wget stuff to file

Skip saving the wget content to files.
Also quite the wget.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index e90303d..7bf0848 100644
--- a/Makefile
+++ b/Makefile
@@ -75,7 +75,7 @@
 	@-repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
 	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(HL_FULL_VER)"
-	@wget http://192.168.0.4:8080/job/goya-release-test/build?token=goya-release
+	@wget -q -O - http://192.168.0.4:8080/job/goya-release-test/build?token=goya-release
 	@echo "Started the \"goya-release-test\" jenkins job... check your email for results."
 
 .PHONY: hl-dali hl-gaudi
